### PR TITLE
WIP add premium subscriber landing page

### PR DESCRIPTION
**Pull request by mcfiredrill**
Mostly just copying what we have in the google doc so far.

### CODE REVIEW

**Review by bradwbradw — APPROVED** [review]

File: app/templates/home/premium.hbs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+50% of proceeds will go directly to the datafruits DJs
+The other 50% will go to maintaining the website and further development
+
+<h3>MEMBER BENEFITS</h3>
+<ul>
+  <li>Get your own profile page on the site</li>
+  <li>Show off your superior status with a cool badge in your profile</li>
+  <li>Become able to use secret emoji / animated emoji</li>
+  <li>Use an animated gif for your profile avatar</li>
+  </li>Vote on new features - everyone’s voice matters</li>
```
Comment (dudethatbe) [inline app/templates/home/premium.hbs:37]: this line starts with and ends with `</li>`  :sweat_smile:

File: app/templates/home/premium.hbs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<h1>DATAFRUITS PREMIUM</h1>
+<h2>Support a listener and DJ owned and ran internet radio station</h2>
+
+$7 / mo - $70 / yearly !
+
+<img src="/assets/images/df_premium.gif" alt="{{ t "premium.membership_card" }}" />
```
Comment (dudethatbe) [inline app/templates/home/premium.hbs:6]: I'm seeing a missing translation error for `premium.membership_card` on the `/premium` page

**Comment by dudethatbe** [comment]
Is there supposed to be a link to the `/premium` page? Maybe it can be merged with the `/subscribe` route? Lots of empty space below that email form :house_with_garden: I can also help out with styling the premium text so it's like the CoC page.

**Review by dudethatbe — CHANGES_REQUESTED** [review]
Guess I could have included my earlier comment in this review text, but I think you get the idea :smile:

**Pull request closed without merging**